internal/plugin/convert/schema.go

```diff
@@ -43,6 +50,15 @@ func ConfigSchemaToProto(b *configschema.Block) *proto.Schema_Block {
 	return block
 }
 
+func protoStringKind(k configschema.StringKind) proto.StringKind {
+	switch k {
+	default:
```

Comment:
the ordering for `default` does not matter unless you use `fallthrough`, i went with `default` because if additional formats were added i wanted it to fall back to `plaintext` if it was unexpected, but i can see the rationale for handling it in other ways. due to this not being mission critical functionality, i'd really prefer not to introduce a `panic`.

Reply:
Can we log an error at least when falling back?